src/main/java/cz/muni/fi/entities/Car.java

```diff
@@ -0,0 +1,130 @@
+package cz.muni.fi.entities;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Class represents car
```

Comment:
Why javadoc description here and nowhere else? I would delete it, it's not really neccessary